docs/user/dashboard/tutorial-create-a-dashboard-of-lens-panels.asciidoc

```diff
@@ -66,296 +97,193 @@ image::images/lens_end_to_end_1_3.png[Changed type and dropped clientip field]
 [role="screenshot"]
 image::images/lens_end_to_end_1_4.png[Flyout config open]
 
-. Click *Save and return*.
-
-[discrete]
-[[view-the-distribution-of-visitors-by-operating-system]]
-=== View the distribution of visitors by operating system
+. Click *Save and return*. You will return to the dashboard, where you can customize
+the newly added panel:
 
-To determine the operating systems you should continue to support, and the importance of mobile traffic from iOS devices, 
-create a donut chart that displays the top operating systems that your visitors used to access your website within the last 90 days.
+.. Drag the bottom corner of the panel until the metric takes up one quarter of the screen
+width. The row for the metric will have 4 items on it later.
 
-. Open *Lens*, then set the <<set-time-filter,time filter>> to *Last 90 days*.
+.. The metric visualization has its own label, so you do not need to add a panel title.
 
-. From the *Chart Type* dropdown, select *Donut*.
+. Click *Save* on the dashboard menu
 
-. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
+. Enter the title "Logs dashboard"
 
-.. Drag and drop *clientip* to the *Size by* field.
-
-.. Drag and drop *machine.os.keyword* to the *Slice by* field.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_2_1_1.png[Donut chart with clientip and machine.os.keyword fields]
-
-. Change the color palette. 
-
-.. In the editor, click *Top values of machine.os.keyword*. 
-
-.. From the *Color palette* dropdown, select *Compatibility*.
-
-.. Click *Close*.
-+
-[role="screenshot"]
-image::images/lens_end_to_end_2_1.png[Donut chart with open config panel]
+. Check the *Store time with dashboard* box, then click *Save*
 
-. Click *Save and return*.
+. After the dashboard refreshes, click *Edit* again.
 
 [discrete]
 [[mixed-multiaxis]]
-=== View the average of bytes transfer per day
+=== View a metric over time
 
-To prevent potential server failures, and optimize the cost of website maintenance, create an area chart that displays the average of bytes transfer. To compare 
-the data to the number of visitors to your website, add a line chart layer.  
+If your index pattern has a default time field, *Lens* will add that field automatically
+when you drag a numeric field onto the workspace panel. This is a helpful shortcut.
+If your data has multiple time fields you can drag to replace the previous time field.
 
-. Open *Lens*.
+For this example, you want to visualize the `bytes` field over time, but you may not have
+chosen a specific function yet:
 
-. From the *Available fields* list, drag and drop *bytes* to the visualization builder.
+. Drag `bytes` from the field list to the workspace panel to have *Lens* automatically
+create a chart. It will create a bar chart using the *median of bytes* as the metric.
```

Comment:
`metric` or `function`?

Reply:
`median of bytes` is a dimension, specifically a metric dimension (as opposed to a bucket dimension). The word "metric" refers to any numeric dimension of a chart, and is a word we use in the Lens table and metric chart types. `median` is the function.

Reply:
I have been referring to `dimension` as `field` because you drag and drop the field onto the editor. Does the field turn into a dimension after Lens parses the field for the visualization? 

Using `metric` is confusing since we have a *Metric* visualization, but I know we use it on the UI as a set of configuration options. Maybe we can change this to:

*Lens* creates a bar chart using the *Median of bytes* as the *Metric configuration*.